src/js/selectors/index.js

```diff
@@ -340,10 +340,9 @@ export const getTenantCapabilities = createSelector(
   ) => {
     const canDelta = isEnterprise || plan === PLANS.professional.id;
     const hasAuditlogs = isAuditlogEnabled && (!isHosted || isEnterprise || plan === PLANS.professional.id);
-    const hasDeviceConfig = hasAddons || (isDeviceConfigEnabled && (!isHosted || addons.some(addon => addon.name === 'configure' && Boolean(addon.enabled))));
-    const hasDeviceConnect =
-      hasAddons || (isDeviceConnectEnabled && (!isHosted || addons.some(addon => addon.name === 'troubleshoot' && Boolean(addon.enabled))));
-    const hasMonitor = hasAddons || (isMonitorEnabled && (!isHosted || addons.some(addon => addon.name === 'monitor' && Boolean(addon.enabled))));
+    const hasDeviceConfig = hasAddons || isDeviceConfigEnabled || addons.some(addon => addon.name === 'configure' && Boolean(addon.enabled));
+    const hasDeviceConnect = hasAddons || isDeviceConnectEnabled || addons.some(addon => addon.name === 'troubleshoot' && Boolean(addon.enabled));
+    const hasMonitor = hasAddons || isMonitorEnabled || addons.some(addon => addon.name === 'monitor' && Boolean(addon.enabled));
```

Comment:
I could be wrong but this change will make "hasDeviceConfig" true on HM independently from the addons status.
Am I right? See: https://github.com/mendersoftware/saas/blob/master/kubernetes/mender_stack/gui/mender-gui.yaml#L87

Reply:
then we can remove these from there I guess...

Reply:
https://github.com/mendersoftware/saas/pull/1069

Reply:
so... if I get it correctly, you use the env variables for OS and rely on add-ons for ENT.
Right?

Reply:
from now on: yes
the only exception being that `HAVE_ADDONS` flag as I'm not sure where it is used (the slack messages from Marcin mentioned the backend tests, but from searching the backend repos I can't find it)...

Reply:
@mzedel `DEVICEAUTH_HAVE_ADDONS` and `USERADM_HAVE_ADDONS` maybe?

Reply:
these might be what became of it - in the messages he referred to `HAVE_ADDONS` in particular and wanted it added to the UI... but I'd also be happy to remove it and see if something would actually fail - it doesn't seem to be used anymore for customer communication

Reply:
`HAVE_ADDONS` should now also be removed.